### PR TITLE
GET Reservation slots by stylist and current day

### PR DESCRIPTION
**Pull request by Siramul**
Added route to get pending and in process reserved slots for a given Stylist on the current day.

### CODE REVIEW

File: express_cut/express_service/serializers.py [hunk 1]
```diff
@@ -196,3 +196,10 @@ class Meta:
 
 
 
+
+
+class AnotherReservationSerializer(serializers.ModelSerializer):
```
Comment (isaacrivriv) [inline express_cut/express_service/serializers.py]: Update name here to something more meaningful

File: express_cut/express_service/urls.py [hunk 2]
```diff
@@ -41,6 +41,7 @@
 
     path('stylist', view_role.get_all_stylists),
     path('stylist/<int:stylist_id>/reservation', view_reservation.reservations_by_stylist),
+    path('stylist/<int:stylist_id>/reservedslots', view_reservation.get_daily_reservations),
```
Comment (isaacrivriv) [inline express_cut/express_service/urls.py]: Lets use camelCase here for the reservedSlots route or simply just go back to reserved should be good as well

File: express_cut/express_service/view_reservation.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import datetime
 
 from .models import User, Reservation, Service
-from .serializers import  ReservationSerializer, DurationSerializer, EstimateSerializer
+from .serializers import ReservationSerializer, DurationSerializer, EstimateSerializer, AnotherReservationSerializer
```
Comment (isaacrivriv) [inline express_cut/express_service/view_reservation.py]: Change name to more meaningful

**Review by isaacrivriv — COMMENTED** [review]
Couple of comments

File: express_cut/express_service/serializers.py (hunk 1 above)
Comment (Siramul) [reply express_cut/express_service/serializers.py]: Will change to ReservationTimeSlotSerializer.

File: express_cut/express_service/urls.py (hunk 2 above)
Comment (Siramul) [reply express_cut/express_service/urls.py]: Will change to reservedTimeSlots.

File: express_cut/express_service/view_reservation.py (hunk 3 above)
Comment (Siramul) [reply express_cut/express_service/view_reservation.py]: Will change.

**Comment by Siramul** [comment]
Made the changes suggested in the comments.

**Review by isaacrivriv — APPROVED** [review]
LGTM Thanks!

**Pull request merged**